### PR TITLE
Log intermediate io separately

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

**Comment by wenleix** [comment]
@rschlussel you should be able to review this :)

File: presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java [hunk 1]
```diff
@@ -62,9 +63,9 @@ public QueryStatistics(
             long totalRows,
             long outputBytes,
             long outputRows,
-            long writtenBytes,
-            long writtenRows,
-            double cumulativeMemory,
+            long writtenOutputBytes,
+            long writtenOutputRows,
+            long writtenIntermediateBytes, double cumulativeMemory,
```
Comment (rschlussel) [inline presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java]: nit: newline

File: presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java [hunk 2]
```diff
@@ -35,8 +35,9 @@
     private final long totalRows;
     private final long outputBytes;
     private final long outputRows;
-    private final long writtenBytes;
-    private final long writtenRows;
+    private final long writtenOutputBytes;
```
Comment (rschlussel) [inline presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java]: nit: these renames should be part of the rename commit

File: presto-product-tests/src/test/resources/com/facebook/presto/tests/querystats/single_query_info_response.json [hunk 3]
```diff
@@ -56,7 +56,9 @@
     "processedInputPositions": 0,
     "outputDataSize": "0B",
     "outputPositions": 0,
-    "physicalWrittenDataSize": "0B",
+    "writtenOutputPositions": 0,
+    "writtenOutputLogicalDataSize": "0B",
+    "writtenOutputPhysicalDataSize": "0B",
```
Comment (rschlussel) [inline presto-product-tests/src/test/resources/com/facebook/presto/tests/querystats/single_query_info_response.json]: you need to add intermediate datasize here too after you add the field.  This is what's causing the test failure

**Pull request merged**